app/models/serializers/work_queue/decision_review_task_serializer.rb

```diff
@@ -77,6 +77,10 @@ def self.veteran(object)
     object[:veteran_participant_id] || veteran(object).participant_id
   end
 
+  attribute :veteran_ssn do |object|
```

Comment:
```suggestion
  object[:veteran_ssn] || attribute :veteran_ssn do |object|
```

What I think would be nice is if `veteran_ssn` was included within the result sets of the task queries over in BusinessLine, similar to [veteran_participant_id](https://github.com/department-of-veterans-affairs/caseflow/blob/ClayS/APPEALS-9132/app/models/organizations/business_line.rb#L127), so that all SSNs could potentially grabbed along with the tasks. This will result in fewer subsequent database calls in the serializer.

Below are before/after photos showing the queries run whenever loading the decision review queue. The `veterans` table queue are cut out in the after screenshot.

Before | After
---|---
![before-ssn-optimization](https://user-images.githubusercontent.com/99351305/215594466-ebd2a901-50b7-4f67-8014-36b4823a673f.png)|![after-ssn-optimization](https://user-images.githubusercontent.com/99351305/215594558-bf8a3def-6a2b-4a55-9e59-1d0372261eae.png)

Reply:
So maybe add SSN to the 'union_select_statements' method in business_line?

Reply:
Exactly!

```ruby
    def union_select_statements
      [
        Task.arel_table[Arel.star],
        issue_count,
        claimant_name_alias,
        participant_id_alias,
        veteran_name_alias,
        "veterans.ssn as veteran_ssn"
      ]
    end    
```

That'll add a `veteran_ssn` column to the output.

I think for it to work with all of the `CONCAT`s it'll need to be added to the `GROUP BY` as well:

```ruby
    def group_by_columns
      "tasks.id, veterans.participant_id, veterans.first_name, veterans.last_name, veterans.ssn, "\
      "unrecognized_party_details.name, unrecognized_party_details.last_name, people.first_name, people.last_name"
    end
```

Reply:
It looks like [Sean is looking](https://github.com/department-of-veterans-affairs/caseflow/pull/17999/files#diff-73f5e2d31d31768ab395502b14c1a753d6933cf7084ec000fc6de463b562539c) to refer to an alias named `veteran_ssn`.